tests/stub/optimizations/test_optimizations.py

```diff
@@ -89,7 +89,7 @@ def double_read(self, mode, new_session, use_tx, routing,
                     res.consume()
                 else:
                     results.append(
-                        list(map(lambda r: r.value, res.next().values))
+                        list(map(lambda r: r.values[0].value, res))
```

Comment:
```suggestion
                        [value.value for rec in res for value in rec.values]
```

Probably not exactly what the test is about, but I still would want to see if the driver for what ever reason decides to send more or less than one value per result.